memory_reference/include/sync_open.py

```diff
@@ -1,14 +1,5 @@
 from contextlib import contextmanager
 
-# @contextmanager
```

Comment:
Please bring back the function and name it `sync_open_rw`. I want to use it for playing around ;-)